stdlib/public/runtime/Errors.cpp

```diff
@@ -188,13 +189,46 @@ void swift::dumpStackTraceEntry(unsigned index, void *framePC,
 #endif
 }
 
+#if defined(__ANDROID__)
+struct UnwindContext {
+  void **current;
+  void **end;
+};
+
+static _Unwind_Reason_Code unwindBacktraceCallback(struct _Unwind_Context *context, void *arg) {
```

Comment:
Up to you on this, but I think that `SwiftUnwindFrame` is more descriptive than `unwindBacktraceCallback` since it unwinds a single frame.